uploader/base.py

```diff
@@ -29,30 +31,40 @@
 app_context = app.app_context()
 app_context.push()
 
+
 db = SQLAlchemy(app)
 ma = Marshmallow(app)
 api = Api(app)
 
 celery_app = Celery("async_video_processor", broker='redis://broker:6379/0')
 
+
 class Status(enum.Enum):
     incomplete = "incomplete"
     uploaded = "uploaded"
     processed = "processed"
 
+
 class Video(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     status = db.Column(Enum(Status))
     uploaded_file_url = db.Column(db.String(255))
-    created_on = db.Column(TIMESTAMP,  
-                        default=datetime.datetime.utcnow) 
-    updated_at = db.Column(TIMESTAMP,  
-                        default=datetime.datetime.utcnow) 
+    created_on = db.Column(TIMESTAMP,
+                           default=datetime.datetime.utcnow)
+    updated_at = db.Column(TIMESTAMP,
+                           default=datetime.datetime.utcnow)
     processed_file_url = db.Column(db.String(255))
 
+
 class VideoSchema(ma.SQLAlchemyAutoSchema):
+    status = EnumField(Status, by_value=True)
+
     class Meta:
-        fields = ('id', 'status', 'uploaded_file_url', 'processed_file_url')
+        model = Video
```

Comment:
Podemos excluir cierta informacion del video 
```suggestion
        model = Video
        exclude = ['uploaded_file_url']
```
Siento que ese campo es relevante para el usuario

Reply:
Siento que es parte del requerimiento.
![image](https://github.com/MISW4204-2024-desarrollo-sw-nube/interna-FPV-drone-racing-league/assets/31115265/f204d625-f9f8-4023-b649-342fc9a0de03)